CHANGELOG.md

```diff
@@ -16,14 +16,14 @@ All notable changes to this project will be documented in this file.
 - Add-scheme-for-userid-hashing
 - Add global mst ([#35](https://github.com/okx/proof-of-reserves-v2/issues/35))
 - Add-merkle-proof-logic
-- Add-merkle-verification-logic
-- Add-merkle-proof-sdk
+- Add-merkle-verification-logic ([#58](https://github.com/okx/proof-of-reserves-v2/issues/58))
```

Comment:
I think this is PR #63

Reply:
![Uploading Screenshot 2024-09-02 at 12.15.00 PM.png…]()

Reply:
<img width="1011" alt="Screenshot 2024-09-02 at 12 15 00 PM" src="https://github.com/user-attachments/assets/433d9ef5-624c-48a3-b661-38e8fbe5919f">